### ISSUE TITLE
[Accessibility] Form elements must have labels

### ISSUE THREAD

**Issue by tysongach** [thoughtbot/administrate#415]
The search input is missing a label.

I ran the [aXe](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) Chrome extension, and it suggests to fix any of the following:

- `aria-label` attribute does not exist or is empty
- `aria-labelledby` attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- Form element does not have an implicit (wrapped) `<label>`
- Element has no `title` attribute or the `title` attribute is empty
